transport.go

```diff
@@ -79,9 +75,9 @@ func New(tr http.RoundTripper, integrationID, installationID int, privateKey []b
 		mu:             &sync.Mutex{},
 	}
 	var err error
-	t.key, err = jwt.ParseRSAPrivateKeyFromPEM(privateKey)
+	t.appsTransport, err = NewAppsTransport(t.tr, t.integrationID, privateKey)
```

Comment:
Why not pass down the `BaseURL` and `Client` here, rather than setting them on each call?

Reply:
It's only useful for people wanting to write unit tests, or if they want to provide a different client, I would expect most people would be fine with the defaults. Making it functional args would be an option, but not worth the hassle.